idea/indexer/src/gear/indexer.ts

```diff
@@ -396,17 +372,19 @@ export class GearIndexer {
     );
   }
 
-  private async indexMissedBlock(number: number, interestedProgram?: string, interestedCode?: string) {
+  private async indexMissedBlock(number: number) {
     logger.warn(`Index missed block ${number}`);
-    return this.indexBlock(number, true, interestedProgram, interestedCode);
+    return this.indexBlock(number, true);
   }
 
   public async indexBlockWithMissedCode(codeId: HexString): Promise<Code | null> {
     const metaStorage = (await this.api.query.gearProgram.metadataStorage(codeId)) as Option<CodeMetadata>;
     if (metaStorage.isSome) {
       const blockNumber = metaStorage.unwrap().blockNumber.toNumber();
 
-      return (await this.indexMissedBlock(blockNumber))[1];
+      await this.indexMissedBlock(blockNumber);
+
+      return await this.tempState.getCode(codeId);
```

Comment:
Because there `await`

https://github.com/gear-tech/gear-js/pull/1261/files#diff-73c6b35b97c2f51439c562ec2c24da1b4e758c3833f9a5224dd38737ff248825R420